build.gradle

```diff
@@ -104,13 +108,21 @@ configure(scalaProjects) {
         compile ('org.scala-lang:scala-library:%scala-version%')
         compile ('org.scala-lang:scala-reflect:%scala-version%')
 
-        testImplementation('junit:junit:4.12')
+        testImplementation(platform("org.junit:junit-bom:$junitBomVersion"))
+        testImplementation("org.junit.vintage:junit-vintage-engine")
+
         testImplementation('org.scalatest:scalatest_%%:3.0.8')
         testImplementation('org.scalamock:scalamock_%%:4.4.0')
         testImplementation('ch.qos.logback:logback-classic:1.1.3')
         testImplementation('org.reflections:reflections:0.9.10')
     }
-    
+
+
+    test{
+        useJUnitPlatform {
+            includeEngines('junit-jupiter', 'junit-vintage')
```

Comment:
Is this what lets us use both Junit 4 style and Junit 5 style?

Reply:
junit-vintage supports Junit4 imports etc.

Reply:
Thanks